PrismSampleApp/PrismSampleApp/ViewModels/ViewAViewModel.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using PrismSampleApp.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(ViewAViewModel))]
+namespace PrismSampleApp.ViewModels
+{
+    public class ViewAViewModel : BindableBase, INavigationAware
```

Comment:
Name the class properly and use Viewmodelbase

Reply:
ViewModelBase has been used here.